scripts/modules/aux_services.py

```diff
@@ -138,3 +138,12 @@ def _content(self):
             labels=None,
             logging=None,
         )
+
+    def isHealthCheckServiceDisabled(self, name):
+        return (
```

Comment:
Nit: Could this be rewritten as `return name in ["opbeans-load-generator" , "filebeat", ... ]` and still achieve what you are looking for? It might read more cleanly.

Reply:
💯